components/ai_chat/resources/page/components/input_box/index.tsx

```diff
@@ -76,10 +78,21 @@ function InputBox(props: InputBoxProps) {
     }
   }
 
+  const querySubmitted = React.useRef(false)
```

Comment:
nit: not strictly required but I'd prefer this to be defined before its used (i.e. before `handleSubmit`)